### DISCUSSION TITLE
Nomination of @holgerfriedrich

### DISCUSSION THREAD

**Discussion by J-N-K** [openhab/openhab-core#4184] [Core Maintainers Discussions]
Dear @openhab/core-maintainers,

I would like to nominate @holgerfriedrich as a core maintainer. He has shown great knowledge of our core eco system (e.g. while upgrading Karaf last time), his code is of good quality and he is always helpful (which he has shown also in other repositories of the openHAB universe).

Please vote here. Thanks!

**Comment by J-N-K** [comment]
@kaikreuzer I think a week is over and we have 3 out of 4 positive votes. According to out rules that is is an approval. Can you add @holgerfriedrich to the maintainers group? An welcome, Holger!

**Comment by kaikreuzer** [comment]
I have added @holgerfriedrich to the group. Big welcome and many thanks for all your great work and commitment, Holger!

**Comment by holgerfriedrich** [comment]
Thanks all for your confidence and thanks for the welcome!
